src/algorithms/local_search/local_search.cpp

```diff
@@ -873,6 +873,9 @@ void LocalSearch<Route,
           continue;
         }
 
+        const auto t_deliveries_sum = _sol[s_t.second].job_deliveries_sum();
```

Comment:
Hmmm, `_current_loads` seems to be a vector of length 0, so `_current_loads[0]` is undefined behaviour. It looks to me like a `RawRoute` is created on that `update_amounts` is never called

Reply:
Thanks for the hint, so this is more of an internal inconsistency than an `Amount`-related problem. There is probably a specific situation with those instances triggering the problem, which would explain it only shows up now and then.

I'll dive into some further debugging!

Reply:
Turns out this function used to be only called upon `Solution` creation on optimized (non-empty) routes. Using it within the local search makes it possible to call it on an empty `RawRoute`. As simple as that.

Now the upside is that since this only shows up on 3 instances, it means that [this filter](https://github.com/VROOM-Project/vroom/blob/990565b465ae270adfc6317a11a3687f9f25e99d/src/algorithms/local_search/local_search.cpp#L877-L882) does a rather good job avoiding Relocate lookups.